tidy3d/components/monitor.py

```diff
@@ -628,6 +628,21 @@ class AbstractFieldProjectionMonitor(SurfaceIntegrationMonitor, FreqMonitor):
         "in the far field of the device.",
     )
 
+    interval_space: Tuple[
+        pydantic.PositiveInt, pydantic.PositiveInt, pydantic.PositiveInt
+    ] = pydantic.Field(
+        (1, 1, 1),
+        title="Spatial interval",
+        description="Number of grid step intervals at which near fields are recorded for "
```

Comment:
So this includes the first and last cell? if there is some discrepancy (the number of cells is not a multiple of `interval_space`), how is it resolved? For example, if there are cells
```
A B C D
```
with interval 2, which will be be included and does it bias to one side (eg. ACD)?

Might be useful to add a brief mention of this if my question makes sense.

Reply:
It's the same as any other usage of `interval_space` - yeah, it would be ACD in your example. I can add a mention of this to the other `interval_space` docstrings too.

Reply:
Does that bias the result towards a spatial dimension? Or break symmetry if it was present in the fields?

Reply:
I think if the result is biased because of this, it means the downsampling is way too coarse already, and likely the result wouldn't be correct even if the points were chosen symmetrically. I guess one issue that could arise is if there is symmetry to the fields like you said, where we expect significant cancellations due to interference of the projected fields coming from the "left" and the "right", but even then there's already no guarantee that our Yee grid will be perfectly symmetric, so as long as the near fields are sampled finely enough, I don't think there should be an issue

Reply:
That's ok was mostly curious. I think the point about the yee cell being already asymmetric makes some sense. If the user expects symmetric results they should apply symmetry to the simulation already.